### PR TITLE
feat: add a function to query nodes info

### PR DESCRIPTION
**Pull request by Smityz**

### CODE REVIEW

File: executor/table_partitions.go [hunk 1]
```diff
@@ -72,3 +72,32 @@ func ShowTablePartitions(client *Client, tableName string) error {
...
+			secNode := client.Nodes.MustGetReplica(sec.GetAddress())
+			secondaries = append(secondaries, secNode.CombinedAddr())
+		}
+		p.SecondariesAddr = strings.Join(secondaries, ",")
+
+		return &p, nil
+	}
+
+	return nil, fmt.Errorf("error partitions")
+}
```
Comment (Shuo-Jia) [inline executor/table_partitions.go:103]: is used for? 

And exist duplicated code, you should refactor it

Comment (Smityz) [reply executor/table_partitions.go:103]: I use this function in my other app, And I think some function in admin-cli can be used as a common function in other app

File: executor/table_partitions.go [hunk 2]
```diff
@@ -44,15 +50,31 @@ func ShowTablePartitions(client *Client, tableName string) error {
...
+		partitions = append(partitions, p)
+	}
+
+	fmt.Println("[PartitionDistribution]")
+	tabular.Print(client, partitions)
+	return nil
+}
+
+func ShowTablePartition(client *Client, tableName string, partitionIndex int32) (p PartitionStruct, err error) {
+	resp, err := client.Meta.QueryConfig(tableName)
```
Comment (Shuo-Jia) [inline executor/table_partitions.go:68]: You re-execute the funtions. you can refactor like as:

```go

ShowTablePartitions(client *Client, tableName string) ([]PartitionStruct, error){}

GetNodePartition(client *Client, tableName string) {
result, _ := ShowTablePartitions()
return results[pid.index]
}
```

**Comment by Smityz** [comment]
I find it too many places neet to refactor, so I will close this PR

**Review by Shuo-Jia — APPROVED** [review]

**Pull request merged**